### DISCUSSION TITLE
What has happened to %MI% in zmfilter.pl

### DISCUSSION THREAD

**Discussion by phils01UK** [ZoneMinder/zoneminder#4106] [Q&A]
Hi

I have just installed a new ZM server, and noticed my scripts all failing - because %MI% for Monitor ID has been removed from zmfilter.pl.

Was there a reason for this? Can I request its added back, or should I just hack my zmfilter.pl to include it.

Thanks

Phil

**Comment by connortechnology** [comment]
Seems to have become %MID%.  I don't recall doing that.  I can certainly add back MI as well.

**Comment by connortechnology** [comment]
I've gone back 8 years and can't find any evidence of support for %MI%.  How old was your install?

**Reply by phils01UK** [reply]
The install is quite old - but not that old I dont think. Accord to mysql my tables were created in 2021.
Even my new install from a month ago does not have eith %MID% or %MI% as filter options.

**Reply by connortechnology** [reply]
Let me guess 1.36 series?  Yeah, doesn't have it.  1.37 does.
